.github/workflows/build-and-publish-release.yaml

```diff
@@ -21,5 +27,13 @@ jobs:
           export_default_credentials: true
       - name: Setup GCR auth
         run: gcloud auth configure-docker --quiet
+      - uses: azure/setup-helm@v1
+        with:
+          version: 'v3.1.2'
+      - uses: broadinstitute/setup-chart-releaser@v1
+        with:
+          version: '1.0.0-beta.1'
       - name: Publish
-        run: sbt publish
+        run: sbt publish reindexHelmRepository
+        env:
+          CR_TOKEN: ${{ secrets.ChartReleaserToken }}
```

Comment:
@snwessel you'll need to make identical changes to the `build-and-publish-release` workflow in encode-ingest as part of initializing the orchestration workflow; it's an unfortunate bit of boilerplate that we can't shove into `sbt`